README.md

```diff
@@ -153,12 +154,24 @@ my-project
 ├── my_project
 │   ├── __init__.py
 │   ├── bar.py
-│   └── my_project.cpython-36m-x86_64-linux-gnu.so
+│   └── _lib_name.cpython-36m-x86_64-linux-gnu.so
 ├── README.md
 └── src
     └── lib.rs
 ```
 
+When doing this also be sure to set the module name in our code to match the Cargo.toml's `lib.name`
```

Comment:
It's actually the last part of `module-name` (split by `.`), `lib.name` in Cargo.toml doesn't matter when using `module-name`.